middleware/src/rpcserver/mocks/Middleware.go

```diff
@@ -68,6 +66,20 @@ func (_m *Middleware) EnableRootLogin(_a0 rpcmessages.ToggleSettingArgs) rpcmess
 	return r0
 }
 
+// EnableSSHPasswordLogin provides a mock function with given fields: _a0
+func (_m *Middleware) EnableSSHPasswordLogin(_a0 rpcmessages.ToggleSettingArgs) rpcmessages.ErrorResponse {
```

Comment:
Why is this part of the diff?

Reply:
I could swear I took it out :o Fixed with a force push.

Reply:
It was never added in https://github.com/digitalbitbox/bitbox-base/pull/237. Apparently `go generate` wasn't there run after renaming the RPCs.

Reply:
Annoying, might make sense to add a check in travis to make sure that go generate does not produce new output.